bentoml/_internal/configuration/containers.py

```diff
@@ -374,6 +388,9 @@ def tracer_provider(
 
         if sample_rate is None:
             sample_rate = 0.0
+            logger.warning(
```

Comment:
👍

Reply:
Looks like this will show up for all users that don't use tracing?

Reply:
@parano You are right, I'm going to change it.

Reply:
Maybe we should warn if  user explicitly set the `sample_rate` to `0.0`. If the `sample_rate` is `Null` we can leave the behavior as-is.

Reply:
@ssheng I still think that we should warn in both situations, since it took us a while to debug why the traces weren't being sent. I'm open to discussing it though.

I just pushed a fix for this, let me know what you think.